community/postsrsd/20-set-socket-listen-address.patch

```diff
@@ -0,0 +1,72 @@
+diff --git a/postsrsd.c b/postsrsd.c
```

Comment:
Please add description to the patch with source of the patch (where you get it) and what does it actually do.